src/Pages/Playground/components/playgrounApiDocs.tsx

```diff
@@ -200,7 +199,9 @@ const Signature: Component<SignatureType> = ({
   buttonClick,
 }) => {
   const [showSignature, setShowSignature] = createSignal(false);
-  const strDescription = description?.replace(/(`.`)/g, match => `<code class="bg-[rgba(255,255,255,.16)] text-[#fff] text-[14px] mr-1.5 px-2 py-1.5 font-normal rounded">${match}</code>`)?.replace(/`/g, '');
+
+  const strDescription = (" " + description?.trim()).replace(/(?<=\s\`)(.*?)(?=`)/g, (match) => `<code class="bg-[rgba(255,255,255,.16)] text-[#fff] text-[14px] mr-1.5 px-2 py-1.5 font-normal rounded">${match}</code>`)?.replace(/`/g, '');
```

Comment:
is this one and the one in the line 56 the same?
If yes we can merge them together

Reply:
yes, I did